services/README.adoc

```diff
@@ -54,38 +54,49 @@ service will be receiving semi-arbitrary blobs of JSON from instances.
-    +------------+   |  | uuid*      <--+    | updatedAt      |
-                     |  | timezone   | ^^    +----------------+
-+----------------+   +--+ channelId  | ||    +--------------------+
-| updates        |   +--+ updateId   | ||    | capabilities       |
-+----------------+   |  | createdAt  | ||    +-------------------- +
-| id             <---+  | updatedAt  | ||    | id                 |
-| sha1           |      +------------+ |+----+ uuid               |
-| manifest (JSON)|                     |     | capabilities (JSON)|
-| createdAt      |                     |     | createdAt          |
-| updatedAt      |                     |     | updatedAt          |
-+----------------+                     |     +--------------------+
-                                       |
-                                       |     +-----------------------+
-                                       |     | versions              |
-                                       |     +-----------------------+
-                                       |     | id                    |
-                                       +-----+ uuid                  |
-                                             | manifest (JSON)       |
-                                             | manifestSchemaVersion |
-                                             | core                  |
-                                             | createdAt             |
-                                             | updatedAt             |
-                                             +-----------------------+
+
+                                                +------------------------+
+                                                | versions               |
+                                                +------------------------+
+                                                |  id                    |
+                                             +--+ *uuid                  |
+                                             |  |  manifest (JSON)       |
+                           +------------+    |  |  manifestSchemaVersion |
+                           | instances  |    |  |  checksum (unique)     |
+                           +------------+    |  |  createdAt             |
+                           |  id        |    |  |  updatedAt             |
+                           | *uuid      <---++  +------------------------+
+                           |  timezone  |   ^^
+                           |  flavor    |   ||
+  +----------------+   +---+  updateId  |   ||
+  | updates        |   |   |  createdAt |   ||
+  +----------------+   |   |  updatedAt |   ||
+  | id             <---+   +------------+   ||
+  | commit         |                        ||
+  | channel        |                        ||  +---------------------+
+  | manifest (JSON)|                        ||  | capabilities        |
+  | tainted        |                        ||  +---------------------+
```

Comment:
as said on the JEP, I wonder if this should be a richer field than just a boolean.